services/app-api/src/resolvers/rate/fetchRate.test.ts

```diff
@@ -352,4 +353,82 @@ describe('fetchRate', () => {
             '2024-02-02'
         )
     })
+
+    // TEST NOT FAILING WHEN EXPECTED, NEED TO FIGURE OUT HOW TO GET RATE REVISIONS OUT OF SYNC
+    it.skip('returns correct linked contractRevisions, even when rate has changed', async()=>{
```

Comment:
I couldn't think of a way to get revisions out of sync. I wonder if anything outside the workflow happened. Feature flagging, migrations, or maybe something in the API silently errored and continued with the request.

Reply:
Thanks for checking it out.  I will delete this test then... but appreciate the eyes and the QA